_config/products.yml

```diff
@@ -79,8 +79,7 @@ devstudio:
         marketplace_url: https://marketplace.eclipse.org/node/1616973
         marketplace_install_url: http://marketplace.eclipse.org/marketplace-client-intro?mpc_install=1616973
         release_date: 2016-04-19
-        # TODO change this to 9.1.0.GA when they release their bits
-        supported_devstudio_is_version: 9.0.0.Beta1
+        supported_devstudio_is_version: 9.0.0.GA
```

Comment:
@pleacu the issue you reported is due to the devstudio version: it must be `9.1.0.GA` instead of `9.1.0.Final`. The `.GA` qualifier is for DevStudio while the `.Final` qualifier is for JBoss Tools.